pipelines/build/common/build_base_file.groovy

```diff
@@ -727,6 +731,7 @@ return {
             enableSigner: Boolean.parseBoolean(enableSigner),
             publish: publish,
             release: release,
+            buildTrigger: releaseType,
```

Comment:
What's the thinking behind have `buildTrigger` and `releaseType` - two differnet names referring to the same thing. That's likely to cause confusion

Reply:
yeah true, I was using the "terminology" already being used in the setTestList() method which mentioned "build trigger".... and releaseType is the existing variable being passed in "Nightly", "Nightly with no publish", "Release"
i'll change to releaseType

Reply:
changed